pom.xml

```
@@ -807,9 +807,66 @@ under the License.</licenseText>
             </dependency>
           </dependencies>
         </plugin>
+        <plugin>
+          <groupId>org.openrewrite.maven</groupId>
+          <artifactId>rewrite-maven-plugin</artifactId>
+          <version>6.8.1</version>
+          <configuration>
+            <activeRecipes>
+              <recipe>org.openrewrite.java.format.AutoFormat</recipe>
+              <recipe>org.openrewrite.java.recipes.JavaRecipeBestPractices</recipe>
```

```suggestion
              <recipe>org.openrewrite.staticanalysis.ReplaceCollectToListWithToList </recipe>
```

- https://github.com/openrewrite/rewrite-static-analysis/pull/558